app/user.go

```diff
@@ -2064,69 +2064,49 @@ func (a *App) RestrictUsersGetByPermissions(userId string, options *model.UserGe
 }
 
 // FilterNonGroupTeamMembers returns the subset of the given user IDs of the users who are not members of groups
-// associated to the team.
-func (a *App) FilterNonGroupTeamMembers(userIDs []string, team *model.Team) ([]string, error) {
+// associated to the team excluding bots
```

Comment:
I noticed that you removed the punctuation from the Godoc, but the convention is to "use correct spelling, punctuation, and sentence structure" as per [Effective Go, Commentary](https://golang.org/doc/effective_go.html#commentary).